auto-complete.js

```diff
@@ -62,6 +62,44 @@
             });
         }
 
+
+    window.changeText = () => {
+            
+        const newText = 
+             `
+             <div style="margin: 0.4em" class="row">
+                <span style="font-size: 10px" type="button" class="chip chip--outline chip--sm" data-toggle="tooltip" data-placement="bottom-right"
+                    data-tooltip ="Identifique as palavras ou termos que, obrigatoriamente, estejam na sua pesquisa.
```

Comment:
Remove this space in other elements too
```suggestion
                    data-tooltip="Identifique as palavras ou termos que, obrigatoriamente, estejam na sua pesquisa.
```